mozilla-release/mobile/android/base/java/org/mozilla/gecko/activitystream/homepanel/topsites/TopSitesPageAdapter.java

```diff
@@ -34,6 +34,10 @@
     private final HomePager.OnUrlOpenListener onUrlOpenListener;
     private final TopPanelRow.OnCardLongClickListener onCardLongClickListener;
 
+    /* Cliqz Start */
+    private boolean isLightTheme;
```

Comment:
`mIsLightTheme`. Are we planning to add a boolean for every other theme we implement or are we using numeric coding?